lix/packages/sdk/src/file-handlers.ts

```diff
@@ -90,28 +87,30 @@ export async function handleFileChange(args: {
 }) {
 	const fileId = args.after?.id ?? args.before?.id;
 
-	const pluginDiffs: any[] = [];
+	const pluginDiffs: {
+		diffs: DiffReport[];
+		pluginKey: string;
+	}[] = [];
 
 	for (const plugin of args.plugins) {
 		// glob expressions are expressed relative without leading / but path has leading /
 		if (!minimatch(normalizePath(args.after.path), "/" + plugin.glob)) {
 			break;
 		}
 
-		const diffs = await plugin.diff!.file!({
+		const diffs = await plugin.diff?.file?.({
```

Comment:
see previous comment about failling instead of optional chaining

Reply:
Will impement this in https://linear.app/opral/issue/LIXDK-107/plugindifffileortype-plugindetectchanges